CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # TimescaleDB Changelog
 
+## 0.7.0 (2017-11-21)
+
+**High-level changes**
+* **Initial PostgreSQL 10 support**. TimescaleDB now should work on both PostgreSQL 9.6 and 10. As this is our first release supporting PG10, we look forward to community feedback and testing.
```

Comment:
We need a note, probably in bold, that says that this update might take a long time.